cmd/tailscale/cli/set.go

```diff
@@ -248,6 +251,26 @@ func runSet(ctx context.Context, args []string) (retErr error) {
 		maskedPrefs.Prefs.RelayServerPort = ptr.To(int(uport))
 	}
 
+	if setArgs.relayServerStaticEndpoints != "" {
+		endpointsMap := map[netip.AddrPort]bool{}
+		endpointsSplit := strings.Split(setArgs.relayServerStaticEndpoints, ",")
+		for _, s := range endpointsSplit {
+			ap, err := netip.ParseAddrPort(s)
+			if err != nil {
+				return fmt.Errorf("%q is not a valid IP:port", s)
+			}
+			endpointsMap[ap] = true
+		}
+		endpoints := make([]netip.AddrPort, 0, len(endpointsMap))
+		for ap := range endpointsMap {
+			endpoints = append(endpoints, ap)
+		}
+		slices.SortFunc(endpoints, func(a, b netip.AddrPort) int {
+			return a.Compare(b)
+		})
```

Comment:
Could be simplified to:
```suggestion
		slices.SortFunc(endpoints, netip.AddrPort.Compare)
```